Makefile

```diff
@@ -99,11 +99,14 @@ build: $(EXECUTABLE)
 
 $(EXECUTABLE): $(SOURCES)
 	go build $(GOFLAGS) $(EXTRA_GOFLAGS) -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)' -o $@
+	
+.PHONY: docker-build
+docker-build:
```

Comment:
I have separate it but still keep it if someone want to build binaries via docker.